docs/api-testing/introduction.md

```diff
@@ -0,0 +1,59 @@
+---
+id: introduction
+title: Using Sauce Labs API Testing
+sidebar_label: About
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## What is API Testing?
+
+An API conversation consists of:
+* API consumer (client side) performing a request
+* API producer (server side) responding to that request<br/><img src={useBaseUrl('img/api-fortress/2022/03/api-conversation.png')} alt="API Conversation and Contract" width="300"/>
+
+The conversation needs to follow specific rules that the API producer and API consumer must agree upon. The formal description of these rules is the contract, which is generally presented as a specification file such as [OpenAPI](https://swagger.io/docs/specification/about/).
+
+If this contract is broken by either party, it can lead to bugs and malfunctions. _Contract testing_ is the act of validating that the API producer and the API consumer are respecting the contract. That’s where Sauce Labs API Testing comes in.
+
+## Contract Testing  
+
+Contract testing is a fast, lightweight form of API testing that strictly checks the content and format of requests and responses. This method is ideal for:
+* Testing APIs during the early stages of design and development
+* Organizations creating APIs that are internal and/or have limited number of consumers
+
+It can also validate spec files in an automated way by capturing how an API consumer and producer communicate with each other. This is typically done in a protected, static environment, where tests are run against mocked (not live) APIs, allowing contract tests to compare isolated API responses to the contract for immediate attention if something is wrong.<br/><img src={useBaseUrl('img/api-fortress/2022/03/api-consumer-contract.png')} alt="API Conversation and Contract" width="500"/>
```

Comment:
@kimsauce This statement "It can also validate spec files in an automated way by capturing how an API consumer and producer communicate with each other." does not make sense, please remove